### PR TITLE
[cpp] add a multiple-root example workspace for test purposes

### PR DESCRIPTION
**Pull request by vince-fugnitto**
#### What it does

The pull-request introduces a new example workspace to test and try out multiple-root support as provided by the `@theia/cpp` extension. The example workspace includes two roots (`program1` and `program2`), the necessary automation scripts, and an easy sample.

#### How to test

Follow the steps provided in the [README](https://github.com/eclipse-theia/theia-cpp-extensions/blob/7c1f319267632984e5243855ed39acc27555c2e0/examples/cpp-multi-root-workspace/README.md)

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

**Comment by lmcbout** [comment]
@vince-fugnitto Following the Use case step in the README file, from step 4, documentation is missing. I see "undefined" in the quick menu for "program 1 " or  "program 2", I don't compile and don't  get the executable,

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto Following the Use case step in the README file, from step 4, documentation is missing. I see "undefined" in the quick menu for "program 1 " or "program 2", I don't compile and don't get the executable,

You must choose the build config, click the `Program1: undefined` and select `A`.

**Comment by lmcbout** [comment]
"Undefined" should not be there first of all

**Comment by vince-fugnitto** [comment]
> "Undefined" should not be there first of all

Why not? It informs users that there is no active build configuration for the given root.

**Comment by vince-fugnitto** [comment]
Ping :)

File: README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ provided by the repository.
 ## Example Workspaces
 - [`cpp-debug-workspace`](./examples/cpp-debug-workspace/README.md)
     - provides an easy and reproducible way to test the functionality present in the `@theia/cpp-debug` extension. Includes a simple C/C++ program, debug launch configuration file (`launch.json`), and a task in order to compile the program (`tasks.json`).
```
Comment (marcdumais-work) [inline README.md]: The mentioned task seems to be missing. Add it?

**Comment by vince-fugnitto** [comment]
Closing the pull-request for the moment. 
The changes are based on `@theia/cpp` which might be deprecated in favor of [`vscode-clangd`](https://open-vsx.org/extension/llvm-vs-code-extensions/vscode-clangd) (which is maintained by the llvm community).

**Pull request closed without merging**